app/src/components/posts/PostListItem.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <v-card class="post mb-3">
+    <v-list-item
+      :ripple="false"
+      three-line
+      class="post-list-item"
+      :class="{ active: active }"
+      @click="onClick"
+    >
+      <v-list-item-content>
+        <v-list-item-title
+          class="headline mb-1"
+          :class="{ 'full-text': active }"
+          v-html="post.title"
+        />
+        <v-list-item-subtitle
+          :set="(distance = postDistance(post))"
+          :class="{
+            'post-subtitle-hidden': active,
+            'post-subtitle': !active,
+          }"
+        >
+          <strong>
+            <span v-if="post.post_struct.location.country === 'Deutschland'">
+              {{ post.post_struct.location.zipcode }}
+            </span>
+            <span> {{ post.post_struct.location.city }} </span>
+            <span v-if="post.post_struct.location.country !== 'Deutschland'">
+              {{ post.post_struct.location.country }}
+            </span>
+            <em
+              v-if="
+                distance && post.post_struct.location.country === 'Deutschland'
+              "
+            >
+              (in {{ distance }})
+            </em>
+          </strong>
+          &mdash;
+          <span v-html="post.task" />
+        </v-list-item-subtitle>
+      </v-list-item-content>
+    </v-list-item>
+    <!-- Detail Card -->
+    <v-card
+      v-if="active && showDetail"
+      class="details-smartphone"
+      :class="{
+        'details-smartphone-visible': active,
+        'details-smartphone-hidden': !active,
+      }"
+    >
+      <v-card-text>
+        <template v-for="el in postTable">
+          <div v-if="post[el.key]" :key="el.key">
+            <h3>{{ el.label }}</h3>
+            <p v-if="el.link">
+              <a :href="post[el.link]" target="_blank">{{ post[el.key] }}</a>
+            </p>
+            <p v-else v-html="post[el.key]" />
```

Comment:
After finding a more descriptive name for postTable, write out the variable el with a descriptive name